spec/Organization.tex

```diff
@@ -137,17 +132,15 @@
 
 \item
 Chapter~\ref{Standard_Modules}, Standard Modules, describes the
-standard modules that are provided with the Chapel language.
+structure of standard modules that are provided with the Chapel language
+and the \chpl{Base} standard module.
```

Comment:
That'd probably be helpful, especially for stuff that's in there and isn't specifically in ChapelBase (which I think @noakesmichael was documenting?)